source/index.html

```diff
@@ -1,12 +1,208 @@
+        </a>
+      </nav>
+    </header>
+    <main class="page-main">
+      <div class="main-title-wrapper">
+        <h1 class="main-title">Милые штуки ручной работы для дома</h1>
+      </div>
+      <section class="categories">
+        <h2 class="visually-hidden">Категории</h2>
+        <!-- Картинки у элементов списка являются декоративными и не меняют цвет => через стили -->
+        <ul class="categories-list">
+          <li class="categories-list__item categories-list__item--interior">
+            <a class="categories-list__link" href="#">Предметы интерьера</a>
+          </li>
+          <li class="categories-list__item categories-list__item--toys">
+            <a class="categories-list__link" href="#">Детские игрушки</a>
+          </li>
+        </ul>
+      </section>
+      <section class="promo">
+        <h2 class="promo__title">Товар недели</h2>
+        <div class="product">
+          <h3 class="section-title">Вязаные корзинки</h3>
+          <p class="product__text">
+            Экологически чистая пряжа, ручная работа. Доступны в шести расцветках и трёх размерах.
+          </p>
+          <table class="product__parameters">
+            <caption class="visually-hidden">Характеристики товара Вязаные корзинки.</caption>
+            <tr>
+              <th>Цвет:</th>
+              <td>«Тиффани»/серый</td>
+            </tr>
+            <tr>
+              <th>Диаметр:</th>
+              <td>25 см</td>
+            </tr>
+            <tr>
+              <th>Высота:</th>
+              <td>15 см</td>
+            </tr>
+          </table>
+          <img  class="product__image" src="img/promo/basket.png" alt="Изображение товара дня магазина Мишка - Вязаные корзинки." width="260" height="315">
+          <p class="product__price">Цена: 990 руб.</p>
+          <button class="button" type="button">Заказать</button>
+        </div>
+      <!-- "Волна" по нижней границе секции является декоративным изображением => через стили -->
+      </section>
+      <section class="advantages">
+        <h2 class="section-title">Коротко о нас:</h2>
+        <!-- Картинки у пунктов списка декоративные, цвет не меняют => через стили -->
+        <ul class="advantages-list">
+          <li class="advantages-list__item advantages-list__item--ecology">Экологически чистые материалы</li>
+          <li class="advantages-list__item advantages-list__item--style">Скандинавский стиль по российской цене</li>
+          <li class="advantages-list__item advantages-list__item--like">Увеличивает лайки на фотографиях</li>
+          <li class="advantages-list__item advantages-list__item--love">Связано вручную с любовью и умилением</li>
+          <li class="advantages-list__item advantages-list__item--domestic">Поддержка отечественного производителя</li>
+          <li class="advantages-list__item advantages-list__item--gift">Поставляется в подарочной упаковке</li>
+        </ul>
+      </section>
+      <section class="reviews slider">
```

Comment:
```suggestion
      <section class="reviews">
```

хватит и одного блока